src/metabase/lib/aggregation.cljc

```diff
@@ -11,7 +11,7 @@
    [metabase.lib.ref :as lib.ref]
    [metabase.lib.schema :as lib.schema]
    [metabase.lib.schema.aggregation :as lib.schema.aggregation]
-   [metabase.lib.schema.external-op :as lib.schema.external-op]
+   [metabase.lib.schema.common :as lib.schema.common]
```

Comment:
I added a schema for external ops in #31590, and then I realized that we already have one. So I removed the schema and consolidated them both into one.

One of the schemas required `:lib/type`, and the other required `:args`; the new consolidated schema requires both for consistency, so I had to update a few functions. The benefit is that now we can just dispatch off of the `:lib/type` key like we do for all the other maps, and don't need to special-case dispatch for external ops anymore.